WordPress/gutenberg
# Fix browser history when synchronising state with urls

closes #48726

## What?

This PRs fixes the browser history when navigating the site editor.

## How?

- It avoids persisting default values in urls (path=/ and canvas=view)
- Prevents unnecessary history changes.

## Testing Instructions

1- Open the site editor
2- Click browser "back" button
3- you should be back in the dashboard properly
4- Testing using Chrome, Firefox or Edge (in trunk it's broken with these)
